README.md

```diff
@@ -103,11 +104,22 @@ if (project.hasProperty("cfLocal")) {
     compileOnly files(cfHome + "/checker/dist/checker-qual.jar")
     testCompileOnly files(cfHome + "/checker/dist/checker-qual.jar")
     checkerFramework files(cfHome + "/checker/dist/checker.jar")
+    // only needed for JDK 8
     checkerFrameworkAnnotatedJDK files(cfHome + "/checker/dist/jdk8.jar")
   }
 }
 ```
 
+### JDK 8 vs JDK 9+
```

Comment:
These are implementation details.  Therefore, I would omit them or state them at the very end of the README, after information that users are more likely to want.

Reply:
I'll move them to the end of the README.